en/modules/openiddict.md

```diff
@@ -0,0 +1,462 @@
+# OpenIddict Module
+
+This module provides integration and management functionality for Identity Server;
+
+* Built on the [OpenIddict-core](https://github.com/openiddict/openiddict-core) library.
+* Manage **Application** and **API resources** in the system. 
+* Set **permissions** for clients.
+* Manage **API resources**
```

Comment:
Not managing API resources as I know. Also, it is duplicated (second item also says it)